### PR TITLE
feat(BE-1998): fork-and-adapt-original-repo

### PR DESCRIPTION
**Pull request by calinblaga**
Signed-off-by: Calin Blaga <calin.blaga@arobs.com>

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,63 +1,26 @@
...
-    "authors": [
-        {
-            "name": "Dominik Zogg",
-            "email": "dominik.zogg@gmail.com"
-        }
-    ],
     "require": {
-        "php": "^7.2",
-        "symfony/process": "^3.4.43|^4.4.11|^5.0"
+        "php": "^8.0",
```
Comment (akuma91) [inline composer.json:7]: shouldn't this be `^8.0|^7.2` or `^8.0|^7.4` ? that way we could also use it with our php projects which aren't on 8 yet, although i think it's only used on media-api. would be worth checking.

File: docker/dev/php/files/php/20-pcov.ini [hunk 2]
```diff
@@ -0,0 +1 @@
+extension=pcov.so
```
Comment (akuma91) [inline docker/dev/php/files/php/20-pcov.ini]: missing nl ^^

**Review by akuma91 — CHANGES_REQUESTED** [review]
two small things, other than that lgtm

File: composer.json (hunk 1 above)
Comment (calinblaga) [reply composer.json:7]: As I see it's only used in media-api, and I don't see it used in other project in the future. And even if so, we'll hopefully have them running on php 8 by then, so I would rather keep it more strict.

**Review by akuma91 — APPROVED** [review]

**Pull request merged**